app/utils/lava_log_parser.py

```diff
@@ -72,11 +75,47 @@ def run(log, boot, txt, html):
     else:
         boot_result_html = "<span class=\"warn\">{}</span>".format(boot_result)
 
+    # panic, error and warn are considered deprecated, but still here 
+    # for ensure that everything will works well in any case.
     formats = {
+        "emerg": "<span class=\"info\">{}</span>\n",
+        "alert": "<span class=\"alert\">{}</span>\n",
+        "crit": "<span class=\"alert\">{}</span>\n",
+        "err": "<span class=\"err\">{}</span>\n",
         "warning": "<span class=\"warn\">{}</span>\n",
+        "notice": "<span class=\"info\">{}</span>\n",
+        "info": "<span class=\"info\">{}</span>\n",
+        "debug": "<span class=\"debug\">{}</span>\n",
+        "panic": "<span class=\"alert\">{}</span>\n",
         "error": "<span class=\"err\">{}</span>\n",
+        "warn": "<span class=\"warn\">{}</span>\n"
     }
-    numbers = {"warning": 0, "error": 0}
+
+    kernel_log_levels = {
+        "0": "<span class=\"info\">{}</span>\n",
+        "1": "<span class=\"alert\">{}</span>\n",
+        "2": "<span class=\"alert\">{}</span>\n",
+        "3": "<span class=\"err\">{}</span>\n",
+        "4": "<span class=\"warn\">{}</span>\n",
+        "5": "<span class=\"info\">{}</span>\n",
+        "6": "<span class=\"info\">{}</span>\n",
+        "7": "<span class=\"debug\">{}</span>\n",
+    }
+
+    numbers = {
+        "emerg": 0, 
+        "alert": 0, 
+        "crit": 0,
+        "err": 0,   
```

Comment:
Having "err" and "error" is a bit confusing here.  Are you mixing kernel log levels with LAVA ones?  Maybe having a different dictionary for kernel levels would make more sense.  You can still then add things together for the total number of errors and warnings in the results:
```python
lava_numbers["error"] + kernel_numbers["err"]
```

Reply:
@gctucker in this cases I was more worried about syslog format in deprecated way (panic, warn and error are considered deprecated (link)[https://en.wikipedia.org/wiki/Syslog#Severity_level]. Do you know if LAVA is still using this format? If you consider that is important, we can count both and write on html file....lava and kernel level cases.

Reply:
The syslog format coming from the kernel log is not related to LAVA, it's directly produced by the kernel.  So it's another separate set of log levels.